server/webapp/WEB-INF/urlrewrite.xml

```diff
@@ -23,6 +23,18 @@
     http://tuckey.org/urlrewrite/
 -->
 <urlrewrite>
+    <rule>
+        <name>Spark Artifact Store Index API</name>
+        <from>^/api/admin/artifact_stores</from>
```

Comment:
Add `(/?)$` at the end.

Reply:
Made this consistent across all Spark APIs and SPAs